test/extended/builds/pipeline.go

```diff
@@ -13,36 +16,98 @@ import (
 	"github.com/openshift/origin/test/extended/util/jenkins"
 )
 
+func debugAnyJenkinsFailure(br *exutil.BuildResult, name string, oc *exutil.CLI, dumpMaster bool) {
+	if !br.BuildSuccess {
```

Comment:
The intent was to just code up the if once vs. coding up the if check in all the places I added calls to `debugAnyJenkinsFailure`.  I can switch it up if you like.